pkg/interface/link/src/js/components/settings.js

```diff
@@ -0,0 +1,301 @@
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import { deSig, uxToHex } from '/lib/util';
+import { Route, Link } from "react-router-dom";
+
+import { LoadingScreen } from './loading';
+import { LinksTabBar } from '/components/lib/links-tabbar';
+import SidebarSwitcher from './lib/icons/icon-sidebar-switch';
+import { makeRoutePath } from '../lib/util';
+
+export class SettingsScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      title: "",
+      description: "",
+      color: ""
+    };
+
+    this.changeTitle = this.changeTitle.bind(this);
+    this.changeDescription = this.changeDescription.bind(this);
+    this.changeColor = this.changeColor.bind(this);
+    this.renderDelete = this.renderDelete.bind(this);
+    this.renderMetadataSettings = this.renderMetadataSettings.bind(this);
+  }
+
+  componentDidMount() {
+    if (this.props.resource) {
+      this.setState({
+        title: this.props.resource.title,
+        description: this.props.resource.description,
+        color: uxToHex(this.props.resource.color || '0x0')
+      });
+    }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const { props, state } = this;
+    if (!!state.isLoading && !props.resource) {
+      this.setState({
+        isLoading: false
+      }, () => {
+        api.setSpinner(false);
+        props.history.push('/~link');
+      });
+    }
+
+    if (props.resource && (prevProps !== props)) {
+      this.setState({
+        title: props.resource.title,
+        description: props.resource.description,
+        color: uxToHex(props.resource.color || '0x0')
+      });
+    }
+  }
+
+  changeTitle() {
+    this.setState({title: event.target.value})
+  }
+
+  changeDescription() {
+    this.setState({description: event.target.value});
+  }
+
+  changeColor() {
+    this.setState({color: event.target.value});
+  }
+
+  deleteCollection() {
+    const { props, state } = this;
+
+    api.deleteCollection(props.resourcePath);
+    api.setSpinner(true);
+
+    this.setState({
+      isLoading: true
+    });
+  }
+
+  renderDelete() {
+    const { props, state } = this;
+
+    const isManaged = ('/~/' !== props.groupPath.slice(0,3));
+
+    let deleteButtonClasses = (props.amOwner) ? 'b--red2 red2 pointer bg-gray0-d' : 'b--grey3 grey3 bg-gray0-d c-default';
+    let leaveButtonClasses = (!props.amOwner) ? "pointer" : "c-default";
+
+    let deleteClasses = 'dib f9 black gray4-d bg-gray0-d ba pa2 b--black b--gray0-d pointer';
+    let deleteText = 'Remove this collection from your collection list.';
+    let deleteAction = 'Remove';
+    if (props.amOwner && isManaged) {
+      deleteText = 'Delete this collection from the group. This deletes it for everyone!';
```

Comment:
Probably just use the phrasing from Chat's settings. Exclamation marks! are not ideal.

Same for below copy too...